node/test/util/synthetic_gc_util.js

```diff
@@ -271,7 +271,9 @@ describe("synthetic_gc_util", function () {
         yield syntheticGcUtil.cleanUpOldRefs(repo, roots, isOlderThanTomorrow);
 
         newSyntheticRefs = yield syntheticGcUtil.getSyntheticRefs(subRootRepo);
-        assert.equal(newSyntheticRefs.toString(), oid2.toString());
+        let EXPECTED_SYNTHETIC_REFS = 1;
```

Comment:
nit: I don't love this variable naming.  First, all-caps vars are typically reserved for constants, not function-level vars.  Second, this variable holds the number of expected synthetic refs, not the refs themselves.  I would rather see let expectedSyntheticRefs = [oid2] and go from there.